<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainForm</class>
 <widget class="QWidget" name="mainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="controleGroupBox">
     <property name="title">
      <string>Steuerung:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnGameStart">
        <property name="text">
         <string>Spiel sta&amp;rten</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnGameBreak">
        <property name="text">
         <string>Spiel abbre&amp;chen</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>208</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnQuit">
        <property name="text">
         <string>Programm&amp;ende</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="settingGroupBox">
     <property name="title">
      <string>Einstellungen:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Spieleranzahl:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinPlayerAmmount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>99</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>384</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="diceGroupBox">
     <property name="title">
      <string>Würfel:</string>
     </property>
     <layout class="QHBoxLayout" name="diceLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Würfe:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdRolleAmmount">
        <property name="digitCount">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRoll">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Würfeln</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="gameFieldGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Spielblatt</string>
     </property>
     <layout class="QGridLayout" name="gameFieldLayout"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnQuit</sender>
   <signal>clicked()</signal>
   <receiver>mainForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
